content/source/docs/cloud/sentinel/examples.html.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: "cloud"
-page_title: "Example Policies - Sentinel - Terraform Cloud"
+page_title: "Example Policies - Policies - Terraform Cloud"
```

Comment:
```suggestion
page_title: "Example Policies - Terraform Cloud"
```

Is this better?

Reply:
It looks better but it kind of moves away from the standard that we seem to be adhering to across all of the other document sections. The current page title that is in use _kind of_ makes sense. Although from a user experience perspective I would probably reverse it because when you hover over the tab in Chrome it is backwards. Not sure if that was intentional but it reads weird to me 🤷‍♂ 

So I would have formatted it like this:
```
page_title: "Terraform Cloud - Policies - Example Policies"
```

Reply:
If there's a standard we may as well stick with it for now. It just read odd when I was reviewing.